### PR TITLE
Update to HiFiCNV v0.1.7

### PR DESCRIPTION
**Pull request by williamrowell**
- updated container tag
- updated paths to expected CN bed files in inputs.*.json and tests (`expected_cn.{hg19,hg38,hs37d5}.{XX,XY}.bed`)
- updated bed files were uploaded to Azure 8/23, so tests will likely fail until files are synced overnight

### CODE REVIEW

**Comment by williamrowell** [comment]
Before release, we need to update static resources on AWS, GCP, and the tarball.

**Comment by williamrowell** [comment]
Depends on https://github.com/PacificBiosciences/wdl-dockerfiles/pull/24

File: backends/aws/inputs.aws.json [hunk 1]
```diff
@@ -53,8 +53,8 @@
       "data": "s3://dnastack-resources/dataset/GRCh38/hificnv/cnv.excluded_regions.common_50.hg38.bed.gz",
       "data_index": "s3://dnastack-resources/dataset/dataset/GRCh38/hificnv/cnv.excluded_regions.common_50.hg38.bed.gz.tbi"
     },
-    "hificnv_expected_bed_male": "s3://dnastack-resources/dataset/dataset/GRCh38/hificnv/male_expected_cn.hg38.bed",
-    "hificnv_expected_bed_female": "s3://dnastack-resources/dataset/dataset/GRCh38/hificnv/female_expected_cn.hg38.bed",
+    "hificnv_expected_bed_male": "s3://dnastack-resources/dataset/dataset/GRCh38/hificnv/expected_cn.hg38.XY.bed",
```
Comment (gconcepcion) [inline backends/aws/inputs.aws.json:56]: theres double dataset/dataset here

File: backends/aws/inputs.aws.json [hunk 2]
```diff
@@ -53,8 +53,8 @@
...
+    "hificnv_expected_bed_female": "s3://dnastack-resources/dataset/dataset/GRCh38/hificnv/expected_cn.hg38.XX.bed",
```
Comment (gconcepcion) [inline backends/aws/inputs.aws.json:57]: dataset/dataset

**Review by gconcepcion — DISMISSED** [review]
other than doubled dataset in the aws inputs, it looks good to me

File: backends/aws/inputs.aws.json (hunk 2 above)
Comment (williamrowell) [reply backends/aws/inputs.aws.json:57]: Nice catch, thanks.

**Pull request merged**